server/schemas/typeDefs.js

```diff
@@ -7,10 +7,11 @@ const typeDefs = `
     availability: Boolean
     role: String
     expertise: String
-    conversations: [Conversation]!
+    conversation: Conversation!
```

Comment:
`conversation` to match model and since each user should only have ONE conversation, it should not be square bracketed